pom.xml

```diff
@@ -527,6 +543,24 @@ limitations under the License.
             </rules>
             </configuration>
           </execution>
+          <execution>
+            <id>check-signatures</id>
+            <phase>test</phase>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <checkSignatureRule implementation="org.codehaus.mojo.animal_sniffer.enforcer.CheckSignatureRule">
+                  <signature>
+                    <groupId>org.codehaus.mojo.signature</groupId>
+                    <artifactId>java16</artifactId>
```

Comment:
using `${javaVersion}` would be more flexible for child projects

Reply:
Changed. However to be useful in child projects, should I change version to 1.0? (version 1.1 of the signatures are only available for java16.)

Reply:
yes, please: I don't know how Java 1.6 signatures 1.0 were broken, but let's hope they work sufficiently
and I suppose we'll upgrade minimum Java version soon, so that will be a question of past :)